Student_t.ipynb

```diff
@@ -0,0 +1,1086 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.      n = norm.pdf(x, loc=0, scale=1)
</pre><p>We should meaningful variable names as mentioned in PEP8 naming convention.</p><p>here n -&gt; norm</p><p>		   l -&gt; laplace</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>